### PR TITLE
Add pre commit and run Black

### PR DESCRIPTION
**Pull request by gcardozo123**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
 install:
   - pip install --upgrade pip setuptools
   - pip install -r dev-requirements.txt
 
-script:
-  - pytest tests.py
\ No newline at end of file
+jobs:
+  include:
+    - script: python -m pytest tests.py
```
Comment (nicoddemus) [inline .travis.yml:10]: I suggest we drop 3.5 and test with 3.7

**Review by nicoddemus — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (gcardozo123) [reply .travis.yml:10]: `f-strings`, here we go! :D

Comment (gcardozo123) [reply .travis.yml:10]: Done

File: err_stash.py [hunk 2]
```diff
@@ -574,62 +674,73 @@ def github_token(self, msg, args):
...
 This only needs to be done once.
 """
 
 NO_GITHUB_TOKEN_MSG = """
-**Github API Token not configured**. 
-Create a new token [here](https://github.com/settings/tokens/new) with **write access** and then execute:
-    `!github token <TOKEN>` 
+**Github API Token not configured**.
+Create a new token [here](https://github.com/settings/tokens/new) with:
+- write access;
```
Comment (nicoddemus) [inline err_stash.py]: What's this?

Comment (gcardozo123) [reply err_stash.py]: Oops, I forgot about this before, it was probably related to Stash token `read/write` access. I removed this part of the message now.

**Review by nicoddemus — APPROVED** [review]

**Pull request merged**